build.gradle

```diff
@@ -46,7 +46,8 @@ subprojects {
         myriadExecutorConf
         myriadExecutorConf.transitive = false
         // exclude hadoop/yarn deps for 'runtime'
-        runtime.exclude group: 'org.apache.hadoop', module: '*'
+        // TODO (Kannan Rajah) Exclude only while packaging
+        //runtime.exclude group: 'org.apache.hadoop', module: '*'
```

Comment:
Fixed in https://github.com/smarella/myriad/commit/ef328d15ffc7fd378fe7ce8417b665d462cbfa72. The unit tests are now executed with ```$PROJECT_HOME/gradlew build```. The hadoop dependencies are excluded from being copied into ```$PROJECT_HOME/myriad-scheduler/build/libs```.